### PR TITLE
Implement developers mode

### LINKED ISSUE #1276: [UI idea] "Development mode"
At one point, kids want to make proper big programs and do not need the tabs anymore, but would rather have more space. Can we make a special "development mode" or something where the whole width of the screen is input and output (so no more grey side bars, and the menu bars using the whole width of the page)

We could maybe allow kids to resize the balance between input and output with a draggable separator (like in https://replit.com/)

(@TiBiBa let me know if this description makes sense!)

### PR DESCRIPTION
**Pull request by TiBiBa**
Provide a general summary of your changes in the Title of the PR

**Description**

In this PR we implement a developers mode to give users the option to hide all "learning" elements that they don't need when focussing just on coding. By pressing the `toggle()` the commands and adventure tabs will be hidden and the code and output window will be increased in width and height.

**Fix for**

This PR fixed #1276.

**How to test**

Press the toggle() button below the "run code" button and notice that all non-coding elements of the page will disappear. Press it again to return to the original view. Notice that the toggle will not be remembered when switching levels.

**Screenshots**
Schermafbeelding 2021-11-15 om 14 05 24
Schermafbeelding 2021-11-15 om 14 05 32

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

### CODE REVIEW

File: templates/incl-editor-and-output.html [hunk 1]
```diff
@@ -106,3 +106,16 @@
...
+<!-- developer mode toggle -->
+<div class="mx-auto w-64 border-gray-400 border-2 rounded-full py-2 px-4">
+  <label for="developers_toggle" class="flex flex-row items-center justify-center cursor-pointer">
+    <div class="relative">
+      <input id="developers_toggle" type="checkbox" onclick="hedyApp.toggle_developers_mode()" class="sr-only" />
+      <div class="w-10 h-4 bg-gray-400 rounded-full shadow-inner"></div>
+      <div class="dot absolute w-6 h-6 bg-white rounded-full shadow -left-1 -top-1 transition" style="top: -5px;"></div>
+    </div>
+    <div class="ml-3">
+      Developers mode
```
Comment (Felienne) [inline templates/incl-editor-and-output.html:118]: Shall we add this to texts and localize it? I guess for Dutch kids this is ok but for Greek or Chinese kids we want the option to show it in their own lang.

**Review by Felienne — CHANGES_REQUESTED** [review]
Super awesome @TiBiBa and so quickly made!

Just want to have the name of the mode localized and we are good to go.

File: templates/incl-editor-and-output.html (hunk 1 above)
Comment (TiBiBa) [reply templates/incl-editor-and-output.html:118]: Will add a localization based on the *.yaml files and will let you know when it's finished.

Comment (TiBiBa) [reply templates/incl-editor-and-output.html]: Fixed!

**Comment by Felienne** [comment]
Awesome @TiBiBa, just changed the wording for kids a bit because they might not know the word developer/ontwikkelaar to mean programmer.

 I just have one tiny request, when in Turtle mode, there now are quite some lines for text, can we limit that to 2 or 3 lines, so there is more room for drawing:

image

**Comment by TiBiBa** [comment]
> Awesome @TiBiBa, just changed the wording for kids a bit because they might not know the word developer/ontwikkelaar to mean programmer.
> 
> I just have one tiny request, when in Turtle mode, there now are quite some lines for text, can we limit that to 2 or 3 lines, so there is more room for drawing:
> 
> image

Fixed this with the last commit!

**Comment by Felienne** [comment]
Kijk eens @OnnoEbbens, u vraagt wij draaien hoor :D Gaat zo live op de site.

**Review by Felienne — APPROVED** [review]

**Pull request merged**